tests/test_aiosource.py

```diff
@@ -56,7 +45,8 @@ async def test_default_warehouse(self):
     @pytest.mark.online
     @pytest.mark.timeout(60)
     @pytest.mark.asyncio
-    async def test_get_packages(self):
+    async def test_get_packages(self):  # type: ignore
```

Comment:
@fridex Only one pre-commit issue we face is this test, to resolve this issue `tests/test_aiosource.py:65:5: F811 redefinition of unused 'test_get_packages' from line 45`, we just can rename this test name because we already have the same name below, but however, if you rename, then this test fails, is it essential keeping this test?
I also tried using await, but still fails
```python
    @pytest.mark.online
    @pytest.mark.timeout(60)
    @pytest.mark.asyncio
    async def test_get_package_hashes(self):
        """Test get packages."""
        source_info = {
            "name": "my-pypi",
            "url": "https://tensorflow.pypi.thoth-station.ninja/index/manylinux2010/AVX2/simple",
            "verify_ssl": True,
            "warehouse": True,
        }

        source = AIOSource.from_dict(source_info)
        package_hashes = await source.get_package_hashes("tensorflow", "2.0.0")
        assert package_hashes == [
            {
                "name": "tensorflow-2.0.0-cp36-cp36m-linux_x86_64.whl",
                "sha256": "9a62e16ea9dc730d006e1271231f318ee2dad48d145fd3b9e902a925ea3cca2e",
            }
        ]
```

Reply:
> @fridex Only one pre-commit issue we face is this test, to resolve this issue `tests/test_aiosource.py:65:5: F811 redefinition of unused 'test_get_packages' from line 45`, we just can rename this test name because we already have the same test name below at line 68, but however, if you rename, then this test still will fail. I also tried modifying it using await, but still fails
> 
> ```python
>     @pytest.mark.online
>     @pytest.mark.timeout(60)
>     @pytest.mark.asyncio
>     async def test_get_package_hashes(self):
>         """Test get packages."""
>         source_info = {
>             "name": "my-pypi",
>             "url": "https://tensorflow.pypi.thoth-station.ninja/index/manylinux2010/AVX2/simple",
>             "verify_ssl": True,
>             "warehouse": True,
>         }
> 
>         source = AIOSource.from_dict(source_info)
>         package_hashes = await source.get_package_hashes("tensorflow", "2.0.0")
>         assert package_hashes == [
>             {
>                 "name": "tensorflow-2.0.0-cp36-cp36m-linux_x86_64.whl",
>                 "sha256": "9a62e16ea9dc730d006e1271231f318ee2dad48d145fd3b9e902a925ea3cca2e",
>             }
>         ]
> ```
> 
> is it essential keeping this test, WDYT?

What is the error output produced? I suspect we have built a new tensorflow so sha does not match. You can adjust it so that hash is not checked directly, but its presence.

Reply:
yeah, looks like, that was the reason, I changed `sha`., thanks!